.github/workflows/publish-go-tester-task.yml

```diff
@@ -35,14 +35,21 @@ jobs:
     outputs:
       result: ${{ steps.determination.outputs.result }}
     steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
       - name: Determine if the rest of the workflow should run
         id: determination
         run: |
           RELEASE_BRANCH_REGEX="refs/heads/[0-9]+.[0-9]+.x"
+          TAG="$(git tag --points-at=HEAD 2> /dev/null | head -n1)"
```

Comment:
```suggestion
```

These changes accompany my suggestion to use `github.ref` to identify `push` event produced by tag push.